lsstswBuild.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
You are being flippant. I was thinking something along the lines of explaining why you are only wrapping the scripts we provide but leaving everything else.